archinstall/default_profiles/desktops/niri.py

```diff
@@ -0,0 +1,80 @@
+from typing import TYPE_CHECKING, override
+
+from archinstall.default_profiles.desktops import SeatAccess
+from archinstall.default_profiles.profile import GreeterType, ProfileType, SelectResult
+from archinstall.default_profiles.xorg import XorgProfile
+from archinstall.tui.curses_menu import SelectMenu
+from archinstall.tui.menu_item import MenuItem, MenuItemGroup
+from archinstall.tui.types import Alignment, FrameProperties, ResultType
+
+if TYPE_CHECKING:
+	from collections.abc import Callable
+
+	from archinstall.lib.translationhandler import DeferredTranslation
+
+	_: Callable[[str], DeferredTranslation]
+
+
+class NiriProfile(XorgProfile):
+	def __init__(self) -> None:
+		super().__init__(
+			'Niri',
+			ProfileType.WindowMgr,
+			description=''
+		)
+
+		self.custom_settings = {'seat_access': None}
+
+	@property
+	@override
+	def packages(self) -> list[str]:
+		additional = []
+		if seat := self.custom_settings.get('seat_access', None):
+			additional = [seat]
+
+		return [
+			"alacritty",
```

Comment:
The wiki recommends these packages to be installed https://wiki.archlinux.org/title/Niri can we stick to that so there's a reference point